app/src/main/java/com/billbreaker/MainActivity.java

```diff
@@ -63,7 +72,18 @@ public void onClick(View view) {
     }
 
     private void populateReceipts() {
-        // receipts = receiptDatabase.getAllReceipts(); TODO uncomment this when ada updates receipt database
+         receipts = receiptDatabase.getAllReceipts();
+//        PersonalReceiptItem shri = new PersonalReceiptItem("shri", 13.34);
+//        PersonalReceiptItem ada = new PersonalReceiptItem("ada", 14.57);
+//        PersonalReceiptItem amrit = new PersonalReceiptItem("amrit", 9.83);
+//        PersonalReceiptItem anna = new PersonalReceiptItem("anna", 11.95);
+//        List<PersonalReceiptItem> items = Arrays.asList(shri, ada, amrit, anna);
+//        Receipt receipt = new Receipt(items, Calendar.getInstance(TimeZone.getTimeZone("UTC")).getTimeInMillis(), 0, 0, 0);
+//
+//        receipts = Arrays.asList(receipt, receipt, receipt);
```

Comment:
same comment as above

Reply:
keeping this in for more testing in the future